python/setup.cfg

```diff
@@ -42,6 +42,16 @@ package_dir =
     = src
 packages = find:
 python_requires = >=3.7
+install_requires =
+    pytz
+    retrying
+    mmh3
+    fastavro>=1.3.2<1.4.0
+    hmsclient==0.1.1
+    boto3
+    pyarrow
+[options.extras_require]
+dev = tox-travis==0.12; pytest
```

Comment:
Can this be formatted just like the `install_requires`?
```
[options.extras_require]
dev =
    tox-travis==0.12
    pytest
```

Reply:
Yes, much nicer!